backend/src/models/contracts/amendments.js

```diff
@@ -4,16 +4,18 @@ const { knex, dataBaseSchemas } = dbConnection();
 const contractAmendmentTable = `${dataBaseSchemas().data}.contract_amendment`;
 const contractsTable = `${dataBaseSchemas().data}.contract`;
 const contractAmendmentTypeTable = `${dataBaseSchemas().data}.amendment_type`;
+const { dateFormatShortYear } = require("@helpers/standards");
 
 // Get all.
 const findAll = (contractId) => {
   return knex
     .columns(
       "contract_amendment.id",
       "contract.co_number as contract",
+
       {
         amendment_date: knex.raw(
-          "TO_CHAR(contract_amendment.amendment_date :: DATE, 'dd-MON-yyyy')"
+          `TO_CHAR(contract_amendment.amendment_date, '${dateFormatShortYear}')`
```

Comment:
Just curious about how some values are being date formatted in the queries but it looks like the frontend is also calling `formatDate()` in the `formConfig`s. Are some dates being formatted in the query and some being formatted by the frontend or is it like all the dates are formatted in the query but the frontend is formatting them again just to make sure?

Reply:
yeah - basically the forms use formConfig, and the tables use tableConfig

and the tables need to be sortable, so i use dateFormatter() there.
the forms don't need to be sortable, so I use formatDate() there.

for some of these models, like this amendments model, I'm changing the format so it's consistent with the dd-Mon-yy format, and also using a constant so it can be changed at a central location later, if necessary

Reply:
Makes sense for table values vs. form values. In the case of form values though it looks like we're formatting twice (once in the query and then again in formConfig)?

Reply:
that could be - I'll take a look tomorrow to see if there's any overlap in the formatting between models and forms. There might be a couple I missed

Reply:
Up to you, it's a lot of places to replace it. I feel like we should be able to get away with not formatting dates in queries and letting the frontend handle it between the new dateFormatter and formatDate(). I guess report queries will need to keep their date formatting or we'd need to figure out Carbone formatting and update all the templates.

Reply:
yeah, when I talked to Shawn about the reports and their date formatting, He said it would be better not to use the Carbone template to format the date, so probably the best place to do it would be in the generic controller - I think I already have that in place, but I'd have to double-check the old reports. 

I agree about removing the format from the models for all reports, forms, and tables though.  It does make sense to have the frontend handle the cosmetic stuff.